### DISCUSSION TITLE
Deactivation on certain pages possible?

### DISCUSSION THREAD

**Discussion by jaguarman** [awcodes/filament-sticky-header#36] [Q&A]
Hi, thanks for the good work.

is it possible to switch sticky headers off on certain resource/pages/wizards.

that would be great

best regards
peter

**Comment by awcodes** [comment]
Currently this isn't possible since the javascript is loaded globally.

**Comment by jaguarman** [comment]
HI Adam,

thanks for your answer,

what about a trait in the page/resource/wizard like i.e.

use noStickyheader;

or

public bool $stickHeader = false;

Best regards
Peter


Am Mo., 11. Nov. 2024 um 14:57 Uhr schrieb Adam Weston <
***@***.***>:

**Reply by awcodes** [reply]
The problem is that the CSS and JS are registered with `FilamentAsset` so it has no context of anything that has to do with the resource.

**Comment by jaguarman** [comment]
Ok, thanks for the clarification.
but it would have been a nice feature ;-)

best regards
Peter


Am Mi., 13. Nov. 2024 um 18:58 Uhr schrieb Adam Weston <
***@***.***>:
